DTML.EduBot/Controllers/MessagesController.cs

```diff
@@ -18,6 +21,14 @@ public class MessagesController : ApiController
         {
             if (activity.Type == ActivityTypes.Message)
             {
+                var userQuery = activity.Text;
+                await MessageTranslator.IdentifyLangAsync(userQuery);
+                var translatedText = await MessageTranslator.TranslateTextAsync(userQuery, "en");
```

Comment:
Since this is a static class, this is not thread safe.

Think about a scenario where the bot is talking in spanish, and after sometime an italian person starts his session. This will change the language for spanish speaker to italian.

To repro:
Use two instances of Bot emulator, and talk in spanish in one and italian in the other. This will result in a race condition.